Userland/Libraries/LibWeb/HTML/Scripting/Environments.cpp

```diff
@@ -326,8 +337,8 @@ JS::Realm& incumbent_realm()
 // https://whatpr.org/html/9893/b8ea975...df5706b/webappapis.html#incumbent-settings-object
 EnvironmentSettingsObject& incumbent_settings_object()
 {
-    // FIXME: Then, the incumbent settings object is the incumbent realm's principal realm settings object.
-    return Bindings::principal_host_defined_environment_settings_object(incumbent_realm());
+    // Then, the incumbent settings object is the incumbent realm's principal realm settings object.
```

Comment:
ignore previous comment on this line about the fixme,  seems fine to resolve here in ` LibWeb: Wire up synthetic realm to settings object and execution context `